pkgs/tools/games/minecraft/mcaselector/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, stdenv
+, fetchurl
+, makeWrapper
+, jre
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "mcaselector";
+  version = "2.2.2";
+
+  src = fetchurl {
+    url = "https://github.com/Querz/mcaselector/releases/download/${finalAttrs.version}/mcaselector-${finalAttrs.version}.jar";
+    sha256 = "sha256-tOSdzLFxvEJ9LXliwfosMkgcrQLsrW7qDS8vrgPzQoI=";
```

Comment:
Prefer `hash = ...` to `sha256 = ...` Both currently work, but the former will make it easier if the algorithm ever needs to change.

Since there could be different hashing algorithms in use, this make sense.